packages/vuetify/src/components/VCalendar/mixins/calendar-with-events.ts

```diff
@@ -126,11 +125,17 @@ export default CalendarBase.extend({
       return this.parsedWeekdays
     },
     categoryMode (): boolean {
-      return false
+      return this.type === 'category'
     },
   },
 
   methods: {
+    eventColorFunction (e: CalendarEvent): string {
```

Comment:
how is this change related to categories?

also `e.color ? e.color : this.eventColor` could be simplified to `e.color || this.eventColor`